style/prettier.json

```diff
@@ -0,0 +1,25 @@
+{
+  "comment": "Disables rules that may conflict with Prettier",
+  "blanks-around-fences": false,
```

Comment:
These rules aren't in conflict, as Prettier adds these too.
It passes the code through https://prettier.io/docs/en/options.html#parser `remark-parse`
Most of the rules below aren't configurable with prettier, but don't conflict with Markdownlint

Reply:
What is the point in having these rules enabled in Markdownlint-land, though? If you have editor extensions enabled for both Markdownlint and Prettier, you'll get reports from both tools about the errors. That seems excessive.

Reply:
I don't use the Prettier tool, so I won't have many opinions about which rules should be listed here. However, I wonder if there is a distinction to be made between what rules are used by the editor plug-in, and what rules are applied on the command-line. In my practice, the CI process is definitive and people can choose to use editor plug-ins, or not. I wouldn't want to set up the rules here such that everyone was required to use both plug-ins in order to write acceptable Markdown.

Reply:
> I wouldn't want to set up the rules here such that everyone was required to use both plug-ins in order to write acceptable Markdown.

Me neither 😄 - I don't think that's what this change would cause. All that's happening is folks who _opt in_ to extending `style/prettier.json` won't have Markdownlint apply them.

A more accurate phrasing from my comment in this thread might have been: If you have _both Markdownlint and Prettier configured_, you'll get reports from both tools about the errors. That seems excessive.

Am I missing something?

Reply:
Makes sense to me! I'll give a day or two for Nick to weigh in if he wants and then merge the PR.

Reply:
It isn't something I'd use/recommend on a project, but I don't think that should block this landing.
Probably need to update the existing documentation though https://github.com/DavidAnson/markdownlint/blob/a852407c887ec60949aa5365ed964bab833f962f/doc/Prettier.md